src/app/components/ListFilter/ListFilter.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ListFilter extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      withJS: false,
+      submitValue: '2018-01-01',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.setState({
+      withJS: true,
+    });
+  }
+
+  handleChange(e) {
+    this.setState(
+      {
+        submitValue: e.target.value,
+      },
+      () => {
+        // this function will be replaced by submiting to endpoint
+        console.log(this.state.submitValue);
+      }
+    );
+  }
+
+  renderFieldset() {
+    const fieldsetProps = this.props.fieldsetProps;
+    const visuallyHidden = (this.state.withJS) ? 'visuallyHidden' : '';
+    const selectName = fieldsetProps.fieldsetName;
+    const selectId = `${selectName}-input`;
+    const defaultValue = (fieldsetProps.options.length && fieldsetProps.options[0].value) ?
```

Comment:
This is fine for now, but if you go to `Spring 2017` directly that won't be the first object in the array so you can't always do `options[0]`.